### DISCUSSION TITLE
Is it possible to add share button for instagram? please 🥹

### DISCUSSION THREAD

**Discussion by wingsterlai** [adityatelange/hugo-PaperMod#1295] [Ideas]
Instagram is the primary social platform that I am using and I would love to share it on Instagram. 
I have tried to search for how to do it and couldn't find an answer. 
It would be nice to add it directly without changing too much code. 
Appreciated it!!

**Comment by adityatelange** [comment]
If you show us 1 website where you have share button for instagram, we can add it
